readme-vars.yml

```diff
@@ -13,17 +13,18 @@ full_custom_readme: |
   docker run --rm \
     -e CONTAINER_NAME=${CONTAINER_NAME} \
     -v ${TEMPDIR}:/ansible/jenkins \
-    linuxserver/jenkins-builder:latest
+    ghcr.io/linuxserver/jenkins-builder:latest
   ```
   **Running locally: **
 
   If you need to test functionality just navigate to the folder with the jenkins-vars.yml and run:
   ```
+  docker rmi ghcr.io/linuxserver/jenkins-builder:latest && \
```

Comment:
why not just pull? this will cause an unnecessary download every time

Reply:
That's a better plan!